src/web/app/pages-instructor/instructor-audit-logs-page/instructor-audit-logs-page.component.html

```diff
@@ -0,0 +1,73 @@
+<h1>Student Activity Logs</h1>
+<p>This page allows you to find when your students have accessed or submitted a particular feedback session.</p>
```

Comment:
Are there plans to include filtering based on feedback session after selecting / filling in the course ID? If yes, then you might want to include a skeleton template for that. If no, then maybe reword this prompt to include "for a given course"

Reply:
Will reword it